docusaurus/docs/learn/quickstarts/network/help/quickstart-walkthrough.md

```diff
@@ -0,0 +1,314 @@
+ZITI_NETWORK_COMPONENTS_IPS="127.0.0.1,127.0.21.71,192.168.100.100"
+```
+
+Now we’ll create the key, client, and server certs for the control plane.
+
+```
+"${ZITI_BIN_DIR}/ziti" pki create key \
+  --pki-root="${ZITI_PKI}" \
+  --ca-name "${ZITI_CTRL_CA_NAME}" \
+  --key-file "${ZITI_NETWORK_COMPONENTS_PKI_NAME}"
+
+"${ZITI_BIN_DIR}/ziti" pki create server \
+  --pki-root="${ZITI_PKI}" \
+  --ca-name "${ZITI_CTRL_CA_NAME}" \
+  --key-file "${ZITI_NETWORK_COMPONENTS_PKI_NAME}" \
+  --server-file "${ZITI_NETWORK_COMPONENTS_PKI_NAME}-server" \
+  --server-name "${ZITI_NETWORK_COMPONENTS_PKI_NAME}-server" \
+  --dns "${ZITI_NETWORK_COMPONENTS_ADDRESSES}" \
+  --ip "${ZITI_NETWORK_COMPONENTS_IPS}"
+
+"${ZITI_BIN_DIR}/ziti" pki create client \
+  --pki-root="${ZITI_PKI}" \
+  --ca-name "${ZITI_CTRL_CA_NAME}" \
+  --key-file "${ZITI_NETWORK_COMPONENTS_PKI_NAME}" \
+  --client-file "${ZITI_NETWORK_COMPONENTS_PKI_NAME}-client" \
+  --client-name "${ZITI_NETWORK_COMPONENTS_PKI_NAME}"
+```
+
+### Create Server and Client Certs for the HTTP API
+
+Set up some initial values for the server and client certificates. As with the control plane, these are for the key file name, DNS, and IP SANs. The DNS and IP lists are reused from the control plane cert generation.
+
+```
+ZITI_EDGE_API_PKI_NAME="ziti.edge.controller"
+ZITI_EDGE_API_ADDRESSES="${ZITI_NETWORK_COMPONENTS_ADDRESSES}"
+ZITI_EDGE_API_IPS="${ZITI_NETWORK_COMPONENTS_IPS}"
+```
+
+Now we’ll create the key, client, and server certs for the HTTP API.
+
+```
+"${ZITI_BIN_DIR}/ziti" pki create key \
+  --pki-root="${ZITI_PKI}" \
+  --ca-name "${ZITI_EDGE_CA_NAME}" \
+  --key-file "${ZITI_EDGE_API_PKI_NAME}"
+  
+"${ZITI_BIN_DIR}/ziti" pki create server \
+  --pki-root="${ZITI_PKI}" \
+  --ca-name "${ZITI_EDGE_CA_NAME}" \
+  --key-file "${ZITI_EDGE_API_PKI_NAME}" \
+  --server-file "${ZITI_EDGE_API_PKI_NAME}-server" \
+  --server-name "${ZITI_EDGE_API_PKI_NAME}-server" \
+  --dns "${ZITI_EDGE_API_ADDRESSES}" \
+  --ip "${ZITI_EDGE_API_IPS}"
+
+"${ZITI_BIN_DIR}/ziti" pki create client \
+  --pki-root="${ZITI_PKI}" \
+  --ca-name "${ZITI_EDGE_CA_NAME}" \
+  --key-file "${ZITI_EDGE_API_PKI_NAME}" \
+  --client-file "${ZITI_EDGE_API_PKI_NAME}-client" \
+  --client-name "${ZITI_EDGE_API_PKI_NAME}"
+```
+
+### Update the CA Bundle
+
+Add the CAs to the CA bundle and make a copy for the HTTP API CA bundle.
+
+```
+cat "${ZITI_PKI}/my.root.ca/certs/my.root.ca.cert" > "${ZITI_PKI}/${ZITI_HOSTNAME}-network-components/cas.pem"
+cat "${ZITI_PKI}/my.root.ca/certs/intermediate.from.external.ca.cert" >> "${ZITI_PKI}/${ZITI_HOSTNAME}-network-components/cas.pem"
+cp "${ZITI_PKI}/${ZITI_HOSTNAME}-network-components/cas.pem" "${ZITI_PKI}/${ZITI_HOSTNAME}-edge/edge.cas.pem"
+```
+
+## Create Controller
+
+### Setup
+
+Declare some environment variables used to generate the controller config file. Some environment variables used were already set previously.
```

Comment:
mmmm. i guess i need to walk back a little bit here. these are all vitally important to the quickstart. explain what each of these variables does in more detail. the short description just isn't enough. i can see a bunch of this doc being reusable in a more "quickstart reference" tpye doc too so don't just throw it all away when you remove it. consider adding it to some kind of page that talks about all the various env vars that can be set and what they do